synapse/lib/stormlib/cortex.py

```diff
@@ -1167,3 +1168,82 @@ async def setHttpApiIndx(self, iden, index=0):
         iden = await s_stormtypes.tostr(iden)
         index = await s_stormtypes.toint(index)
         return await self.runt.snap.view.core.setHttpApiIndx(iden, index)
+
+class StormPoolSetCmd(s_storm.Cmd):
+    '''
+    Setup a Storm query offload mirror pool for the Cortex.
+    '''
+    name = 'cortex.storm.pool.set'
+    def getArgParser(self):
+        pars = s_storm.Cmd.getArgParser(self)
+        pars.add_argument('--connection-timeout', type='int', default=2,
+            help='The maximum amount of time to wait for a connection from the pool to become available.')
+        pars.add_argument('--sync-timeout', type='int', default=2,
+            help='The maximum amount of time to wait for the mirror to be in sync with the leader')
+        pars.add_argument('url', type='str', required=True, help='The telepath URL for the AHA service pool.')
+        return pars
+
+    async def execStormCmd(self, runt, genr):
+
+        if not self.runtsafe: # pragma: no cover
+            mesg = 'cortex.storm.pool.set arguments must be runtsafe.'
+            raise s_exc.StormRuntimeError(mesg=mesg)
+
+        if not self.runt.isAdmin(): # pragma: no cover
+            mesg = 'cortex.storm.pool.set command requires global admin permissions.'
+            raise s_exc.AuthDeny(mesg=mesg)
```

Comment:
```suggestion
            raise s_exc.AuthDeny(mesg=mesg, user=self.runt.user.iden, username=self.runt.user.name)
```

Reply:
Good call, converting to `self.runt.reqAdmin()` :+1: